integration_tests/test_pv_connection.py

```diff
@@ -36,7 +36,7 @@ def on_market_cycle(self, market_info):
                 assert listed_offer["energy"] == listed_offer["energy"]
                 # Try to delete the offer
                 delete_resp = self.delete_offer(offer_info["id"])
-                assert delete_resp["deleted_offer"] == offer_info["id"]
+                assert delete_resp["deleted_offers"] == [offer_info["id"]]
```

Comment:
Same here, Colin should be notified for any change in the endpoint response structure